packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -75,16 +75,26 @@ spec:
     mixer:
```

Comment:
Should prob wrap this, as if both policy and telemetry is off it'd be empty

Reply:
added!